contrib/scripts/localnet-uniswap.sh

```
@@ -0,0 +1,166 @@
+#!/bin/bash
+set -e
+
+# Set localnet settings
+BINARY="nibid"
+CHAIN_ID="nibiru-localnet-0"
+MNEMONIC="guard cream sadness conduct invite crumble clock pudding hole grit liar hotel maid produce squeeze return argue turtle know drive eight casino maze host"
```

Shellcheck (0.10.0)

[warning] MNEMONIC appears unused. Verify use (or export if used externally).

(SC2034)